packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -11,6 +11,21 @@ import {
     ITree,
 } from "@fluidframework/protocol-definitions";
 
+/**
+ * Represents routes from a Fluid object to other Fluid objects in the Container. This is used for garbage collection.
+ */
+export interface IFluidObjectReferences {
```

Comment:
I'd call it a "GC node". its path property is a route, and routes represents outbound connections.
But that thing combined is node, it's not "references". It represents single node for GC system

Reply:
I went back and forth with this (naming is hard!). The reason I didn't add garbage collection or GC phrase to this name is because I wanted to keep it independent of GC. The data stores and channel contexts simply return a set of its references. Then a root entity will use this and pass it to the GC process as an IGarbageCollectionNode. How about IReferenceGraphNode - It is used to build a reference graph of fluid objects in this document?

Having said that, I don't have very strong feeling towards it and calling it IGarbageCollectionNode is fine with me :-).

Reply:
Changed it to IGraphNode. The variables / members are called gcNodes.